chain/chain/src/chain.rs

```diff
@@ -2451,6 +2455,7 @@ impl Chain {
 /// and decide to accept it or reject it.
 /// If rejected nothing will be updated in underlying storage.
 /// Safe to stop process mid way (Ctrl+C or crash).
+#[allow(unused)]
```

Comment:
Can we put unsued on a specific field and/or add a comment? Might help with noticing when the unused annotation is itself unused.